cmd/vela-terraform/apply_test.go

```diff
@@ -59,9 +60,13 @@ func TestTerraform_Apply_Command(t *testing.T) {
 		fmt.Sprintf("-var-file=%s", a.VarFiles[1]),
 	)
 
-	got := a.Command()
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("Command is %v, want %v", got, want)
+	got := a.Command(t.Context())
+	if got.Path != want.Path {
+		t.Errorf("Command path is %v, want %v", got.Path, want.Path)
+	}
+
+	if !reflect.DeepEqual(got.Args, want.Args) {
```

Comment:
suggestion: since `.Args` is just a slice of string, we could switch to `slices.Equals()`; here and in the other places.